### ISSUE TITLE
CSRF blocked the API request

### ISSUE THREAD

**Issue by wangqian0306** [wurmlab/sequenceserver#809]
Hi everyone, I found a fundamental question. The official documentation states that you can use the API to call SequenceServer. I also tried the recommended sequenceserver-remote-blast-python-api library, but I encountered CSRF interception.

I don't know about Ruby, but I asked GPT and saw that in [code](https://github.com/wurmlab/sequenceserver/blob/master/lib/sequenceserver/routes.rb#L69)
it seems that CSRF detection is permanently enabled. Can we make this a configuration item? Or is there another API to get CSRF Token?

**Comment by tadast** [comment]
You're right, this is a bug, sorry. We might need to make CSRF protection configurable as a temporary measure. While we're working on it, you should be able to comment out/delete the line you've linked to disable it.

**Comment by yannickwurm** [comment]
Hi @wangqian0306  - we'd be delighted for a pull request that makes this an optional setting

**Comment by wangqian0306** [comment]
@yannickwurm Is this correct? https://github.com/wurmlab/sequenceserver/pull/810
